bin/build_containers.sh

```diff
@@ -46,7 +46,7 @@ elif grep -qF "Raspberry Pi 3" /proc/device-tree/model || [ "${BUILD_TARGET}" ==
 elif grep -qF "Raspberry Pi 2" /proc/device-tree/model || [ "${BUILD_TARGET}" == 'pi2' ]; then
     export BOARD="pi2"
     export BASE_IMAGE=balenalib/raspberry-pi2
-    export TARGET_PLATFORM=linux/arm/v6
+    export TARGET_PLATFORM=linux/arm/v7
```

Comment:
I'm fairly sure this is wrong.

Reply:
Got it, I'm curious that I get `armv7l` when I run `uname -m` on a Raspberry Pi 2 B running 32-bit RPi OS.

Reply:
I already reverted the changes on this one since ARMv7 is backward compatible.

Reply:
Just read your recent comment. Interesting, indeed. It shows `armv7l` on both RPi OS and balenaOS instances (for Pi 2).
For Pi 1, it shows something like `armv6`. Don't have a personal Pi 1 yet, so I'm not able to check it out for the time being.

Let me take a closer look again on those balena instances.

Reply:
@vpetersson, @rusko124
Here's what I got from a Pi 1 running balenaOS:

![image](https://user-images.githubusercontent.com/10234135/228657138-3a18c7d2-e5ad-4055-bb10-6525b65f87c9.png)

Here's what I got for Pi 2:

![image](https://user-images.githubusercontent.com/10234135/228657485-e61dd5fd-fd78-46b7-abfd-d40e908034cb.png)

Reply:
Here's some balenaOS logs (`anthias-wifi-connect` only):

![image](https://user-images.githubusercontent.com/10234135/228698639-14b7b2fb-3484-419c-9c86-489e14ebbbaf.png)